src/uct/tcp/tcp.h

```diff
@@ -270,6 +274,7 @@ struct uct_tcp_ep {
     uint8_t                       ctx_caps;         /* Which contexts are supported */
     int                           fd;               /* Socket file descriptor */
     uct_tcp_ep_conn_state_t       conn_state;       /* State of connection with peer */
+    unsigned                      conn_attempts;    /* Number of connection attempts done */
```

Comment:
maybe move 'conn_state' after the field 'ctx_caps' will reduce structure size

Reply:
unfortunately, it doesn't help